<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteCustomer</class>
 <widget class="QDialog" name="DeleteCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>614</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>614</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Delete Customer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: rgb(52, 52, 52);
}
QGroupBox
{
	color: rgb(191, 191, 191);
	font-size: 18px;
}
QLabel
{
	color: rgb(191, 191, 191);
	font-size: 17px;
}
QLineEdit
{	
	color: rgb(255, 255, 255);
	border-style: solid;
	background-color: rgb(30, 30, 30);
	font-size: 17px;
}
QPushButton
{
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(255, 255, 255);
	font-size: 18px;
}
QPushButton:hover
{
	background-color: rgb(255, 255, 255);
	color: rgb(52, 52, 52);
}
QRadioButton
{
	font-size: 17px;
	color: rgb(191, 191, 191);
}
QComboBox
{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	font-size: 17px;
}
QComboBox:hover
{
	background-color: rgb(0, 0, 0);
}
QComboBox QAbstractItemView
{
	background-color: rgb(52, 52, 52);
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>471</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_deleteCustomer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Customer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>51</y>
      <width>391</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_showName">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_showUsername">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_customerType">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Customer Type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_showCustomerType">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
